docs/nightly/en/greptimecloud/integrations/mysql.md

```diff
@@ -36,7 +36,7 @@ Use following connect string for your JDBC client.
 jdbc:mysql://<host>:4002/<dbname>?user=<username>&password=<password>
 ```
 
-Read the [time zone setting](https://docs.greptime.com/user-guide/client-libraries/java#time-zone) for JDBC.
+Read the [time zone setting](https://docs.greptime.com/user-guide/ingest-data/for-iot/grpc-sdks/java#time-zone) for JDBC.
```

Comment:
Since we moved client-libraries to ingest-data, there is no place for the MySQL JDBC section in the Java SDK document now, as the content is belong to the query scope.
I'll comment out this link, and add it back when complete the SQL tools document.